surveys/contextual-survey-metadata.json

```diff
@@ -1,21 +1,34 @@
 [
     {
-        "uniqueId": "test_channel",
-        "url": "https://google.qualtrics.com/jfe/form/SV_5gsB2EuG5Et5Yy2",
+        "uniqueId": "eca0100a-505b-4539-96d0-57235f816cef",
```

Comment:
Using a UUID generator here for the uniqueId instead of the previous test string